internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -229,6 +232,11 @@ func (connector *SingleUseConnector) Connect(
 		connInterceptor,
 		connectionResultChan)
 
+	// CALLOUT: this was one of the sources of the jdbc issues
+	// NOTE: this means that we pass on the client login request and call it a day
+	// the rest of the auth negotiation takes place within the duplex pipe
+	return backendConn, err
+
 	if err != nil {
 		connector.writeErrorToClient(err)
 		return nil, err
```

Comment:
The call to connector.WriteLoginResponse on line 245 needs to be deleted, I believe?